packages/accounts-base/accounts_client.js

```diff
@@ -438,4 +453,24 @@ if (Package.blaze) {
   Package.blaze.Blaze.Template.registerHelper('loggingIn', function () {
     return Meteor.loggingIn();
   });
+
+  /**
+   * @global
+   * @name  loggingOut
+   * @isHelper true
+   * @summary Calls [Meteor.loggingOut()](#meteor_loggingout).
+   */
+  Package.blaze.Blaze.Template.registerHelper('loggingOut', function () {
```

Comment:
Just reviewing this now - it looks like there is a [`Package.blaze` check just above the helper registration code](https://github.com/hwillson/meteor/blob/9b8f5a87864e2e6399964957bbeef32b8ca5606f/packages/accounts-base/accounts_client.js#L436). Do you think this is defensive enough? I can expand this further by converting the `Package.blaze` check into a `Package.hasOwnProperty('blaze')` check, paired with some additional checks to make sure the blaze object contains a `Blaze` property, but this might be overkill (I think it's pretty safe to assume `Package.blaze` really only points to a valid blaze object, if truthy). Let me know though - thanks!